yente/data/dataset.py

```diff
@@ -41,6 +45,8 @@ def __init__(self, catalog: DataCatalog["Dataset"], data: Dict[str, Any]):
                     ts = datetime_iso(mdt)
             self.version = iso_to_version(ts) or "static"
 
+        self.delta_index = data.get("delta_url", None)
```

Comment:
Maybe self.delta_url? That might help keep it more explainable... the metadata is a mess anyways.